db/warehouse/migrate/20240531020035_migrate_hmis_form_data.rb

```diff
@@ -0,0 +1,20 @@
+class MigrateHmisFormData < ActiveRecord::Migration[7.0]
+  def up
+    raise unless Hmis::Form::FormProcessor.column_names.include?('backup_values')
+    raise unless Hmis::Form::Definition.column_names.include?('backup_definition')
+
+    # Rake::Task.clear # clear previously loaded tasks
+    # Rails.application.load_tasks # reload tasks from the application
+
+    Rake::Task['driver:hmis:migrate_hmis_link_ids'].invoke
```

Comment:
I did encounter an error locally on an inability to transform the link id `"0354bf9d-f658-4689-9e55-8fcc2058f252"`. I believe it's because the regex says that it should not start with a number, but `transform_identifier` does not add the q_ prefix in this case

Reply:
Was able to run successfully with this change:

```diff
-    # value is just a number
-    value = value =~ /\A[0-9]+\z/ ? "q_#{value}" : value
+    # value starts with a number
+    value = value =~ /\A[0-9]/ ? "q_#{value}" : value
```

Reply:
Great catch, that regex was too strict. Thanks!